stdlib.go

```diff
@@ -70,17 +81,19 @@ func NewStdlibAdapter(logger Logger, options ...StdlibAdapterOption) io.Writer {
 		timestampKey: "ts",
 		fileKey:      "caller",
 		messageKey:   "msg",
+		logRegexp:    StdlibRegexpFull,
```

Comment:
This is inconsistent with documentation.

Reply:
Fixed